tests/tensorflow/nightly/perfzero.libsonnet

```diff
@@ -0,0 +1,275 @@
+local common = import "common.libsonnet";
+local timeouts = import "templates/timeouts.libsonnet";
+local tpus = import "templates/tpus.libsonnet";
+local gpus = import "templates/gpus.libsonnet";
+local utils = import "templates/utils.libsonnet";
+
+{
+  local PerfZeroTest = common.ModelGardenTest {
+    local config = self,
+
+    timeout: timeouts.one_hour,
+    schedule: "0 8 * * *",
```

Comment:
Just confirming this is a good time to catch the new nightly builds. It's the same time we use for the default `func` tests: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/mixins.libsonnet#L22

If you see flakiness due to resource shortages, could be worth trying to stagger and see if it has any effect.

Reply:
This is what we use for our current TensorFlow suite, but these tests should be much shorter. I will adjust timing if necessary.